www/front_src/src/Resources/columns/State/DetailsTable/index.tsx

```diff
@@ -27,9 +27,13 @@ const getYesNoLabel = (value): string => (value ? labelYes : labelNo);
 
 const columnMaxWidth = 150;
 
+interface DetailsTableColumn extends Column {
+  getContent: (details) => string | JSX.Element;
```

Comment:
I didn't want to break the current Listing component implementation.
We'll need to  update the manner of how the columns are managed (maybe only component columns is enough)